accord-core/src/main/java/accord/local/Command.java

```diff
@@ -1240,7 +1251,7 @@ public boolean equals(Object o)
             if (!super.equals(o)) return false;
             Executed executed = (Executed) o;
             return Objects.equals(writes, executed.writes)
-                && Objects.equals(result, executed.result);
+                   && Objects.equals(result, executed.result); // TODO: find a different way to check/assert?
```

Comment:
why this todo?  what is the issue with the check?

Reply:
`TODO` should be `//TODO (classification 1, classification 2, etc.): msg`

This is so we can find them easier and find the *why* and *when* to fix issues.

Reply:
the main classifications are priority ones, and should go _first_:

desired
expected
required

Reply:
Reverted this; this was a tmp TODO.